From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Alfred Neumayer <dev.beidl@gmail.com>
Date: Mon, 4 Jul 2022 14:33:20 +0200
Subject: [PATCH] (halium) e2fsck & misc: Build some components for recovery
 ramdisk

Allows e2fsck & tune2fs to be included in the recovery.

Co-authored-by: TheKit <nekit1000@gmail.com>
Change-Id: I668c4440b9d3825eb279741b6250f09473e0f4d1
---
 e2fsck/Android.bp | 1 +
 misc/Android.bp   | 1 +
 2 files changed, 2 insertions(+)

diff --git a/e2fsck/Android.bp b/e2fsck/Android.bp
index 166d765..dc608e7 100644
--- a/e2fsck/Android.bp
+++ b/e2fsck/Android.bp
@@ -64,6 +64,7 @@ e2fsck_libs = [
 cc_binary {
     name: "e2fsck",
     host_supported: true,
+    recovery_available: true,
     vendor_ramdisk_available: true,
     defaults: ["e2fsck-defaults"],
 
diff --git a/misc/Android.bp b/misc/Android.bp
index ca4267b..a368724 100644
--- a/misc/Android.bp
+++ b/misc/Android.bp
@@ -141,6 +141,7 @@ tune2fs_libs = [
 cc_binary {
     name: "tune2fs",
     host_supported: true,
+    recovery_available: true,
     vendor_ramdisk_available: true,
     defaults: ["tune2fs-defaults"],
 
-- 
2.39.1

